var/spack/repos/builtin/packages/zee/package.py

```diff
@@ -46,6 +49,7 @@ class Zee(CMakePackage):
 
     depends_on('metis+int64')
     depends_on('petsc +int64', when='+petsc')
+    depends_on('timemory ~caliper~cuda~cupti~gperftools~papi~python~mpi', when='+timemory')
```

Comment:
IMO the spec should not contain these variants. They should be declared in the deployment configuration instead.

Reply:
ok, I have moved the variants away.